.dockerignore

```diff
@@ -0,0 +1,7 @@
+node_modules
+.dockerignore
+Dockerfile
+npm-debug.log
+.git
+.hg

```

Comment:
Don't need this - and we should add `tests`, since those don't need to be bundled in.

Reply:
I did a copy paste from somewhere for node ;). I remvoved

Reply:
There's also probably a few other files that should be ignored. But I think as a first version it is ok.